engine/language-server/src/main/scala/org/enso/languageserver/search/SuggestionsHandler.scala

```diff
@@ -201,6 +201,10 @@ final class SuggestionsHandler(
         initialized(projectName, graph, clients - client.clientId, state)
       )
 
+    case msg: Api.SuggestionsDatabaseSuggestionsLoadedNotification
```

Comment:
Why are we changing something related to `SuggestionsDatabaseSuggestionsLoadedNotification`? How is the serialization of modules related to notifications?